web/app/js/components/Namespace.jsx

```diff
@@ -125,8 +128,8 @@ class Namespaces extends React.Component {
 
   render() {
     const {metrics} = this.state;
-    let noMetrics = _.isEmpty(metrics.pod);
-    let deploymentsWithMetrics = _.filter(metrics.deployment, "requestRate");
+    let noMetrics = _isEmpty(metrics.pod);
+    let deploymentsWithMetrics = _filter(metrics.deployment, d => d.requestRate > 0);
```

Comment:
I think native JS has a `.filter` function, would it be worthwhile using that here?

Reply:
there's a number of filters throughout the codebase; this could be a good follow up ticket!